### PR TITLE
feat: Added blood type info to patient

### LINKED ISSUE #2186: Add blood type info to patient
## Scenario
**AS**
A user who can see patient details

**I WANT**
Blood type information displayed

**SO THAT**
I can be aware of someone's blood type

## Acceptance Criteria

> **GIVEN**
> A patient needs to be created
> 
> **WHEN**
> I create the patient
> 
> **THEN**
> An option for blood type should be shown

> **GIVEN**
> A patient needs to be edited
> 
> **WHEN**
> I edit the patient
> 
> **THEN**
> An option for blood type should be shown

> **GIVEN**
> A patient
> 
> **WHEN**
> I view the patient
> 
> **THEN**
> The blood type should be shown

## Technical Notes
Possible blood types: 
A+
A-
AB+
AB-
B+
B-
O+
O-

### PR DESCRIPTION
**Pull request by saksham93**
Added blood type info to patient

Fixes #2186 .

1. Added blood type field in patient 
2. Added Blood Type in General Information
3. Added required translations

**Newly added dependencies with [Bundlephobia](https://bundlephobia.com/) links:**
-  No new dependencies added.

### CODE REVIEW

File: src/patients/GeneralInformation.tsx [hunk 1]
```diff
@@ -145,6 +157,16 @@ const GeneralInformation = (props: Props): ReactElement => {
...
+          <div className="col">
+            <SelectWithLabelFormGroup
+              name="bloodType"
+              label={t('patient.bloodType')}
+              options={bloodTypeOptions}
+              defaultSelected={bloodTypeOptions.filter(({ value }) => value === patient.bloodType)}
+              onChange={(values) => onFieldChange('bloodType', values[0])}
+              isEditable={isEditable}
+            />
+          </div>
```
Comment (jackcmeyer) [inline src/patients/GeneralInformation.tsx:169]: let's add a test in `GeneralInformation.test.tsx` that it renders this select component. Let's also add a test that if the value of the component changes, that it calls the onChange function with the correct data.

Comment (saksham93) [reply src/patients/GeneralInformation.tsx:169]: @jackcmeyer Thank you for your feedback. I have added the test cases.

**Comment by saksham93** [comment]
@tehKapa Just a clarification, I have completed changes on my end. Is the "In progress" label for review under progress or development under progress?
Thank you.

**Comment by tehKapa** [comment]
Review under progress ;)

**Comment by saksham93** [comment]
@jackcmeyer - Can you please review this PR again if the changes have been met? Thank you.

**Pull request merged**